cli/cmd/dashboard.go

```diff
@@ -55,6 +74,26 @@ var dashboardCmd = &cobra.Command{
 	},
 }
 
+func isDashboardAvailable(kubeAPI k8s.KubernetesApi) (bool, error) {
```

Comment:
Can we add a test for this function?

Reply:
Yep I can definitely do that, sorry for dropping the ball on that.